envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/assertions/EnvoyAssertions.kt

```diff
@@ -87,9 +87,12 @@ private fun ObjectAssert<EnvoyContainer>.hasOneAccessDenial(
 }
 
 private fun ObjectAssert<String>.matchesRbacAccessDeniedLog(logDescription: RbacLogDescription) = satisfies {
-    assertThat(it).isEqualTo(
-        "$RBAC_LOG_PREFIX: method = ${logDescription.method}, path = '${logDescription.path}', " +
-            "clientIp = ${logDescription.clientIp}, clientName = ${logDescription.clientName}, " +
-            "protocol = ${logDescription.protocol}, statusCode = ${logDescription.statusCode}\n"
-    )
+    val logLine = "$RBAC_LOG_PREFIX { " +
```

Comment:
How about using a multiline string literal?
https://kotlinlang.org/docs/reference/basic-types.html#string-literals

Reply:
IMO it wouldn't look so great in this case. This string is only one line, so we would need to wrap every line in three quotes, e.g.:
```kotlin
""""method": "${logDescription.method}", """ +
```

Reply:
You are right, I've missed that it's only one line.